handlebars/README.md

```
@@ -49,8 +49,8 @@ package main
 import (
 	"log"
 	
-	"github.com/gofiber/fiber/v2"
-	"github.com/gofiber/template/handlebars/v2"
+	"github.com/gofiber/fiber/v3"
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)